src/platform/plugins/shared/discover/public/application/main/components/top_nav/app_menu_actions/get_share.tsx

```diff
@@ -153,6 +156,7 @@ export const getShareAppMenuItem = ({
       onClose: () => {
         anchorElement?.focus();
       },
+      totalHits: totalHitsState.result || 0,
```

Comment:
Let's pass in `totalHits` as a property on `sharingData`, rather than a new value to the share context especially that this doesn't apply to all share functionality

Reply:
Resolved in this commit, https://github.com/elastic/kibana/pull/242603/commits/6788e70ef5ee7c08c06b4734d52b4a345e4b2c4c